src/Tasks/Microsoft.NET.Build.Tasks/SingleProjectInfo.cs

```diff
@@ -92,9 +118,21 @@ public static SingleProjectInfo Create(string projectPath, string name, string f
                 SingleProjectInfo referenceProjectInfo;
                 if (projectReferences.TryGetValue(sourceProjectFile, out referenceProjectInfo))
                 {
-                    ResourceAssemblyInfo resourceAssemblyInfo =
-                        ResourceAssemblyInfo.CreateFromReferenceSatellitePath(projectReferenceSatellitePath);
-                    referenceProjectInfo._resourceAssemblies.Add(resourceAssemblyInfo);
+                    string originalItemSpec = projectReferenceSatellitePath.GetMetadata("OriginalItemSpec");
```

Comment:
Extract to MetadataKeys.

Reply:
Not sure what you are asking for here.

Reply:
ah, I see, there is a static class which contains constants for these strings.

Reply:
Fixed.